2_deploy_ses_aio/prepare-ses-ansible.yml

```diff
@@ -6,6 +6,8 @@
     ses_ansible_folder: "{{ playbook_dir }}/../ses-ansible/"
     ses_ip: "{{ lookup('file', playbook_dir + '/../.ses_ip') }}"
   tasks:
+    # The role and play expect the hostname to be 'ses' in the
```

Comment:
Thanks for adding this - I will also add it to my notes for using the playbooks on manually-installed SES/CAASP.